source/MRMesh/MRObjectLines.h

```diff
@@ -41,57 +27,17 @@ class MRMESH_CLASS ObjectLines : public VisualObject
     { return reinterpret_cast< const std::shared_ptr<const Polyline3>& >( polyline_ ); } // reinterpret_cast to avoid making a copy of shared_ptr
```

Comment:
there is such accessor in ObjectLinesHolder class, so no need of it here

Reply:
removed